README.md

```diff
@@ -120,7 +120,9 @@ $ ./upload-asset.py  \
 {'url': u'http://localhost:8017/v1/xxxxx-MY-IMAGE.png', 'image': True, 'created': u'Tue Sep 27 16:13:22 2016', 'file_path': u'xxxxx-MY-IMAGE.png', 'tags': u''}
 ```
 
-You can also use [curl](https://curl.haxx.se/docs/manpage.html):
+
+** USING THE METHOD BELLOW SEEMS TO UPLOAD CORRUPT FILES **
```

Comment:
If this is the case and we don't have a way to fix it, just remove this and ask them to use the `upload-assets` snap instead.